styles/theme.conf.tsx

```diff
@@ -1,26 +1,46 @@
 import { extendTheme } from '@chakra-ui/react';
+import { mode } from '@chakra-ui/theme-tools';
 
 const theme = extendTheme({
   styles: {
-    global: {
-      'html, body': {
-        lineHeight: 'tall',
-        bg: `url('/images/bg.svg')`,
-        color: '#FFF'
+    global: (props: Record<string, any>) => ({
```

Comment:
Add an initial color mode with 'light and 'dark' keys.
https://chakra-ui.com/docs/styled-system/color-mode#updating-the-theme-config